test/util/e2e-helpers.js

```diff
@@ -37,8 +37,8 @@ function overrideVetsGovApi(client) {
 }
 
 module.exports = {
-  baseUrl: 'http://localhost:3333',
-  apiUrl: 'http://localhost:4000',
+  baseUrl: `http://localhost:${process.env.WEB_PORT || 3333}`,
+  apiUrl: `http://localhost:${process.env.API_PORT || 4000}`,
```

Comment:
What is `process.env.API_PORT`? I wonder if this might be the cause of the HCA test failures? Our mock endpoint might be different from vets-api one because we're still in healthcare-application?